linkerd/app/src/lib.rs

```diff
@@ -100,21 +103,31 @@ impl Config {
 
         let tap = info_span!("tap").in_scope(|| tap.build(identity.local(), drain_rx.clone()))?;
         let dst = {
-            use linkerd2_app_core::{classify, control, reconnect, transport::tls};
+            const EWMA_DEFAULT_RTT: Duration = Duration::from_millis(30);
+            const EWMA_DECAY: Duration = Duration::from_secs(10);
```

Comment:
What are sensible values for all these constants and magix numbers when it comes to communicating with the control plane?